website/docs/Examples/AutoML-NLP.md

```diff
@@ -26,8 +26,8 @@ automl = AutoML()
 automl_settings = {
```

Comment:
In the end, add a link to the notebook like other examples, such as
[Link to notebook](https://github.com/microsoft/FLAML/blob/main/notebook/flaml_automl.ipynb) | [Open in colab](https://colab.research.google.com/github/microsoft/FLAML/blob/main/notebook/flaml_automl.ipynb)